alpakaConfig.cmake

```diff
@@ -534,8 +534,8 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                             MESSAGE(FATAL_ERROR "NVCC 10.0 does not support clang 7+. Please use clang 4, 5 or 6!")
                         ENDIF()
                     ELSEIF(CUDA_VERSION VERSION_EQUAL 10.1)
-                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 8.0)
-                            MESSAGE(FATAL_ERROR "NVCC 10.1 does not support clang 8+. Please use clang 4, 5, 6 or 7!")
+                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 9.0)
+                            MESSAGE(FATAL_ERROR "NVCC 10.1 does not support clang 9+. Please use clang 4, 5, 6, 7 or 8!")
```

Comment:
That support has actually been added in CUDA 10.1 Update 1 - aka CUDA 10.1.168, not in the original CUDA 10.1 release https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

Reply:
Yes, I know. I do not know a way to differentiate those two versions within CMake.

Reply:
Does CUDA_VERSION not include a patch level?